README.md

```diff
@@ -1,10 +1,217 @@
 [![Build Status](https://travis-ci.org/openmrs/openmrs-esm-patient-chart.svg?branch=master)](https://travis-ci.org/openmrs/openmrs-esm-patient-chart)
 
-Openmrs ESM Patient Chart
-=========
+# Openmrs ESM Patient Chart
```

Comment:
OpenMRS?

Reply:
(not sure here - sometimes we use something like `openmrs-esm-patient-chart`, but here it seems we want it in title case)

Reply:
```suggestion
# OpenMRS ESM Patient Chart
```